### ISSUE TITLE
Directional Coupler

### ISSUE THREAD

**Issue by smartalecH** [SiEPIC/SiEPIC_EBeam_PDK#187]
I noticed the Directional Coupler specs, as designed in the toolbox, are different from your paper's results.

For example. the paper recommends that L1 and L2 for a TE waveguide should be 12.4 and 4.6 microns respectively. The toolbox appears to use 33 and 0.55 microns respectively.

Is there any particular reason for the difference? The documentation has a broken link where "more information" is provided.

**Comment by lukasc-ubc** [comment]
Hi @smartalecH,

Sorry I didn't notice your post earlier.  Can you let me know which device you are referring to specifically, and where the link is missing?

**Comment by smartalecH** [comment]
Yes, the [broadband directional coupler](https://github.com/lukasc-ubc/SiEPIC_EBeam_PDK/wiki/Component-Library-description#broadband-directional-coupler) (specifically the TE mode device).

The wiki says the following link should provide more info:

https://github.com/lukasc-ubc/SiEPIC_EBeam_PDK/tree/master/Documentation/Broadband_DC
